charts/home-assistant/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0]
```

Comment:
It's probably bad practice to retroactively modify changelog/release notes for a previous version, but maybe we should add something like this to the `postgresql` upgrade in 3.0.0?

`Any pre-existing StatefulSet will have to be removed before upgrading due to a name change in the chart.`